apps/frontend/src/app/components/layout/header.component.ts

```diff
@@ -49,7 +49,7 @@ export class HeaderComponent implements OnInit {
   protected unreadNotificationCount = 0;
 
   constructor(
-    private readonly localUserService: LocalUserService,
+    protected readonly localUserService: LocalUserService,
```

Comment:
I prefer a method in this class that calls this, rather than exposing a service to a template